<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddInStkDlg</class>
 <widget class="QDialog" name="AddInStkDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>660</width>
    <height>406</height>
   </rect>
  </property>
  <property name="sizeIncrement">
   <size>
    <width>66</width>
    <height>40</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>660</width>
    <height>406</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="TitleBarWidget" name="titleWidget" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>40</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>40</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="mainWidget" native="true">
     <property name="styleSheet">
      <string notr="true">QWidget#mainWidget{
        background-color:#505050;
}</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="leftMargin">
       <number>20</number>
      </property>
      <property name="topMargin">
       <number>2</number>
      </property>
      <property name="rightMargin">
       <number>20</number>
      </property>
      <property name="bottomMargin">
       <number>2</number>
      </property>
      <item>
       <widget class="QWidget" name="widget_2" native="true">
        <property name="minimumSize">
         <size>
          <width>284</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>284</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="sizeIncrement">
         <size>
          <width>28</width>
          <height>36</height>
         </size>
        </property>
        <property name="baseSize">
         <size>
          <width>284</width>
          <height>362</height>
         </size>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QWidget" name="saveTopWidget" native="true">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>129</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>64</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QWidget#saveTopWidget{
        background-color:#fafafa;
}</string>
           </property>
           <widget class="QLabel" name="goodsNameLabel">
            <property name="geometry">
             <rect>
              <x>18</x>
              <y>5</y>
              <width>231</width>
              <height>16</height>
             </rect>
            </property>
            <property name="font">
             <font>
              <family>微软雅黑</family>
             </font>
            </property>
            <property name="text">
             <string>商品名称：</string>
            </property>
           </widget>
           <widget class="QLabel" name="goodsCodeLabel">
            <property name="geometry">
             <rect>
              <x>18</x>
              <y>28</y>
              <width>231</width>
              <height>16</height>
             </rect>
            </property>
            <property name="font">
             <font>
              <family>微软雅黑</family>
             </font>
            </property>
            <property name="text">
             <string>商品编号：</string>
            </property>
           </widget>
           <widget class="QLabel" name="goodsKindLabel">
            <property name="geometry">
             <rect>
              <x>18</x>
              <y>51</y>
              <width>231</width>
              <height>16</height>
             </rect>
            </property>
            <property name="font">
             <font>
              <family>微软雅黑</family>
             </font>
            </property>
            <property name="text">
             <string>      大类：</string>
            </property>
           </widget>
           <widget class="QLabel" name="goodsUnitLabel">
            <property name="geometry">
             <rect>
              <x>18</x>
              <y>74</y>
              <width>231</width>
              <height>16</height>
             </rect>
            </property>
            <property name="font">
             <font>
              <family>微软雅黑</family>
             </font>
            </property>
            <property name="text">
             <string>规格单位：</string>
            </property>
           </widget>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="saveMainWidget" native="true">
           <property name="styleSheet">
            <string notr="true">QWidget#saveMainWidget{
        background-color:#ffffff;
}</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <item>
             <widget class="QFrame" name="mainFrame">
              <widget class="QLabel" name="label_2">
               <property name="geometry">
                <rect>
                 <x>20</x>
                 <y>15</y>
                 <width>61</width>
                 <height>20</height>
                </rect>
               </property>
               <property name="font">
                <font>
                 <family>微软雅黑</family>
                 <pointsize>12</pointsize>
                </font>
               </property>
               <property name="text">
                <string>单价：</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
              </widget>
              <widget class="QLabel" name="label">
               <property name="geometry">
                <rect>
                 <x>20</x>
                 <y>55</y>
                 <width>61</width>
                 <height>21</height>
                </rect>
               </property>
               <property name="font">
                <font>
                 <family>微软雅黑</family>
                 <pointsize>12</pointsize>
                </font>
               </property>
               <property name="text">
                <string>数量：</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
              </widget>
              <widget class="QLabel" name="label_5">
               <property name="geometry">
                <rect>
                 <x>20</x>
                 <y>85</y>
                 <width>61</width>
                 <height>31</height>
                </rect>
               </property>
               <property name="font">
                <font>
                 <family>微软雅黑</family>
                 <pointsize>12</pointsize>
                </font>
               </property>
               <property name="text">
                <string>件数：</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
              </widget>
              <widget class="QLabel" name="label_6">
               <property name="geometry">
                <rect>
                 <x>-11</x>
                 <y>130</y>
                 <width>91</width>
                 <height>31</height>
                </rect>
               </property>
               <property name="font">
                <font>
                 <family>微软雅黑</family>
                 <pointsize>12</pointsize>
                </font>
               </property>
               <property name="text">
                <string>小计金额：</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
              </widget>
              <widget class="QLineEdit" name="priceLedit">
               <property name="geometry">
                <rect>
                 <x>90</x>
                 <y>10</y>
                 <width>170</width>
                 <height>31</height>
                </rect>
               </property>
               <property name="styleSheet">
                <string notr="true">font-size:14px;
color: rgb(102, 102, 102);
padding-left:10px;</string>
               </property>
               <property name="text">
                <string>0.00</string>
               </property>
              </widget>
              <widget class="QLineEdit" name="countLedit">
               <property name="geometry">
                <rect>
                 <x>90</x>
                 <y>50</y>
                 <width>170</width>
                 <height>31</height>
                </rect>
               </property>
               <property name="styleSheet">
                <string notr="true">font-size:14px;
color: rgb(102, 102, 102);
padding-left:10px;</string>
               </property>
               <property name="text">
                <string>0.00</string>
               </property>
              </widget>
              <widget class="QLineEdit" name="boxNumLedit">
               <property name="geometry">
                <rect>
                 <x>90</x>
                 <y>90</y>
                 <width>170</width>
                 <height>31</height>
                </rect>
               </property>
               <property name="styleSheet">
                <string notr="true">font-size:14px;
color: rgb(102, 102, 102);
padding-left:10px;</string>
               </property>
               <property name="text">
                <string>0.00</string>
               </property>
              </widget>
              <widget class="QLabel" name="subTotalLb">
               <property name="geometry">
                <rect>
                 <x>90</x>
                 <y>130</y>
                 <width>170</width>
                 <height>31</height>
                </rect>
               </property>
               <property name="palette">
                <palette>
                 <active>
                  <colorrole role="Button">
                   <brush brushstyle="SolidPattern">
                    <color alpha="255">
                     <red>250</red>
                     <green>250</green>
                     <blue>250</blue>
                    </color>
                   </brush>
                  </colorrole>
                  <colorrole role="Base">
                   <brush brushstyle="SolidPattern">
                    <color alpha="255">
                     <red>250</red>
                     <green>250</green>
                     <blue>250</blue>
                    </color>
                   </brush>
                  </colorrole>
                  <colorrole role="Window">
                   <brush brushstyle="SolidPattern">
                    <color alpha="255">
                     <red>250</red>
                     <green>250</green>
                     <blue>250</blue>
                    </color>
                   </brush>
                  </colorrole>
                 </active>
                 <inactive>
                  <colorrole role="Button">
                   <brush brushstyle="SolidPattern">
                    <color alpha="255">
                     <red>250</red>
                     <green>250</green>
                     <blue>250</blue>
                    </color>
                   </brush>
                  </colorrole>
                  <colorrole role="Base">
                   <brush brushstyle="SolidPattern">
                    <color alpha="255">
                     <red>250</red>
                     <green>250</green>
                     <blue>250</blue>
                    </color>
                   </brush>
                  </colorrole>
                  <colorrole role="Window">
                   <brush brushstyle="SolidPattern">
                    <color alpha="255">
                     <red>250</red>
                     <green>250</green>
                     <blue>250</blue>
                    </color>
                   </brush>
                  </colorrole>
                 </inactive>
                 <disabled>
                  <colorrole role="Button">
                   <brush brushstyle="SolidPattern">
                    <color alpha="255">
                     <red>250</red>
                     <green>250</green>
                     <blue>250</blue>
                    </color>
                   </brush>
                  </colorrole>
                  <colorrole role="Base">
                   <brush brushstyle="SolidPattern">
                    <color alpha="255">
                     <red>250</red>
                     <green>250</green>
                     <blue>250</blue>
                    </color>
                   </brush>
                  </colorrole>
                  <colorrole role="Window">
                   <brush brushstyle="SolidPattern">
                    <color alpha="255">
                     <red>250</red>
                     <green>250</green>
                     <blue>250</blue>
                    </color>
                   </brush>
                  </colorrole>
                 </disabled>
                </palette>
               </property>
               <property name="styleSheet">
                <string notr="true">font-size:14px;
background-color: rgb(250, 250, 250);
padding-left:10px;</string>
               </property>
               <property name="text">
                <string/>
               </property>
              </widget>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="saveButton">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>42</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>42</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">QPushButton{
color:#ffffff;
font-size:16px;
border-image: url(:/loginImage/commonButton.png);
}

QPushButton:pressed{
border-image: url(:/loginImage/commonButtonPressed.png);
}</string>
              </property>
              <property name="text">
               <string>保存</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="WndDigitalKeyboard" name="keyboardWidget" native="true">
        <property name="minimumSize">
         <size>
          <width>294</width>
          <height>294</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>294</width>
          <height>294</height>
         </size>
        </property>
        <property name="sizeIncrement">
         <size>
          <width>29</width>
          <height>29</height>
         </size>
        </property>
        <property name="baseSize">
         <size>
          <width>294</width>
          <height>294</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>WndDigitalKeyboard</class>
   <extends>QWidget</extends>
   <header>WndDigitalKeyboard.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>TitleBarWidget</class>
   <extends>QWidget</extends>
   <header>TitleBarWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
